testsuite/tests/singlecluster/limitador/tokens_rate_limit/test_trlp.py

```diff
@@ -0,0 +1,142 @@
+"""Test TokenRateLimitPolicy functionality"""
+
+import pytest
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador]
+
+
+BASIC_REQUEST = {
+    "model": "meta-llama/Llama-3.1-8B-Instruct",
+    "messages": [{"role": "user", "content": "What is Kubernetes?"}],
+}
+
+
+@pytest.mark.parametrize("token_rate_limit", ["route", "gateway"], indirect=True)
```

Comment:
You don't need indirect parametrization in these tests. It is used to granularly set parameters for each test, but you have the same line on every test, which we trying to avoid. Less code. 
```py
@pytest.fixture(scope="module", params=["route", "gateway"])
```
on trlp fixture or something similar will be enough